### ISSUE TITLE
detailslist - accessibility labels for sortAscending, sortDescending, filter , group are not added when used onRenderColumnHeaderTooltip

### ISSUE THREAD

**Issue by arunarokk** [microsoft/fluentui#8035]
### Environment Information 
 
- __Package version(s)__: 6.116.2 
- __Browser and OS versions__: Edge, win 10

#### Actual behavior:
 
accessibility labels for sortAscending, sortDescending, filter , group are not added when used onRenderColumnHeaderTooltip.

looking at the codebase https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.base.tsx
_renderAccessibleLabel - there is check for onRenderColumnHeaderTooltip, labels are added only when this onRenderColumnHeaderTooltip doesn't  available on props.
 
#### Expected behavior:
 
these accessibility labels should be added when used onRenderColumnHeaderTooltip

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (No)
 
Requested priority: ( Normal) 
 
Products/sites affected: (Power Apps portal)

**Comment by aneeshack4** [comment]
I think this is related to https://github.com/OfficeDev/office-ui-fabric-react/issues/2786.

**Comment by aneeshack4** [comment]
@KevinTCoughlin This is another DetailsList issue - it looks like a dupe of #2786 because that one also talks about aria labels. Do you know why these aria labels aren't added in the case of onRenderColumnHeaderTooltip?

**Comment by KevinTCoughlin** [comment]
> @KevinTCoughlin This is another DetailsList issue - it looks like a dupe of #2786 because that one also talks about aria labels. Do you know why these aria labels aren't added in the case of onRenderColumnHeaderTooltip?

Hrm its not clear to me how #2786 is related. @arunarokk can you clarify the ask here perhaps with the DOM markup you expect given a condition?

Is there an accessibility validation error that you can share?

**Comment by arunarokk** [comment]
filterAriaLabel,
sortAscendingAriaLabel,      
sortDescendingAriaLabel,
when we pass these aria labels on IColumn, we have these labels are created in the DOM, 
Ex: <label id="header5-column2-tooltip" class="accessibleLabel-95">Sorted Z to A</label>

Without onRenderColumnHeaderTooltip: https://codepen.io/anon/pen/yrGwKa

however, when we use the onRenderColumnHeaderTooltip used these labels are not being added to DOM.
With onRenderColumnHeaderTooltip: https://codepen.io/anon/pen/YdXbQR

@aneeshack4 , please consider this and reopen this bug?

**Comment by KevinTCoughlin** [comment]
cc: @natalieethell @Vitalius1 @aditima we may want to consider this during List backlog review
